docs/pipeline.rst

```diff
@@ -173,6 +173,18 @@ intent_classifier_sklearn
     to other classifiers it also provides rankings of the labels that did not "win". The spacy intent classifier
     needs to be preceded by a featurizer in the pipeline. This featurizer creates the features used for the classification.
 
+intent_featurizer_regex
```

Comment:
I know, I was thinking about changing it to `featurizer_regex` either one has its advantages. what do you think, rename?